DataHub/Integration/01_ManageAttributes.md

```diff
@@ -0,0 +1,228 @@
+
+
+
+## Delete an attribute
+
+You can move an attribute to the *DELETED ATTRIBUTES* tab if it is no longer needed.
+Attributes that are assigned as defining attributes to any variant set cannot be deleted. As there are usually dependencies on an attribute, for example through an attribute set or created products, it is strongly recommended not to delete an attribute.
+Instead, you can deactivate an attribute and thus prevent its use, see [Deactivate an attribute](#deactivate-an-attribute).
+
+#### Prerequisites
+
+At least one attribute has been created, see [Create an attribute](#create-an-attribute).
+
+#### Procedure
+
+*DataHub > Settings > Tab ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/DataHub/Settings/Attributes/Attributes.png "[Attributes]")
+
+1. Select the checkbox of the attribute you want to delete in the list of attributes.   
+  The editing toolbar is displayed above the attributes list.
+
+2. Click the ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete) button in the editing toolbar.    
+  The *Confirm deletion of attribute?* window is displayed.
+
+  ![Confirm deletion of attribute](../../Assets/Screenshots/DataHub/Settings/Attributes/ConfirmDeletionAttribute.png "[Confirm deletion of attribute]")
+
+  > [Info] All dependencies to attribute sets, ETL mappings and entity entries are displayed in this window. Be aware that problems may occur if you delete an attribute with existing dependencies. For defining attributes, the [CONFIRM] button is not displayed in the *Confirm deletion of attribute?* window as they cannot be deleted.   
+
+3. Click the [CONFIRM] button in the bottom right corner.   
+  The attribute has been deleted and is moved to the *DELETED ATTRIBUTES* tab. The *Confirm deletion of attribute?* window is closed.
+
+  > [Warning] Be aware that problems may occur if you delete an attribute with existing dependencies.
+
+> [Info] The attribute is moved to the *DELETED ATTRIBUTES* tab, but it can still be recovered. To irretrievably delete the attribute, see [Permanently delete an attribute](#permanently-delete-an-attribute).
+
+
+
+## Recover an attribute
+
+If you have moved an attribute to the *DELETED ATTRIBUTES* tab, it is not irretrievably deleted. You can still recover the attribute and make the deletion undone.
+
+#### Prerequisites
+
+At least one attribute has been moved to the *DELETED ATTRIBUTES* tab, see [Delete an attribute](#delete-an-attribute).
+
+#### Procedure
+
+*DataHub > Settings > Tab DELETED ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/DataHub/Settings/DeletedAttributes/DeletedAttributes.png "[Attributes]")
+
+1. Select the checkbox of the attribute you want to recover in the list of deleted attributes.   
+  The editing toolbar is displayed above the deleted attributes list.
+
+2. Click the [RECOVER] button in the editing toolbar.   
+  The selected attribute has been recovered and is removed from the list of deleted attributes.
+
```

Comment:
Hinzufügen als Result? 
The recovered attribute is moved back to the *ATTRIBUTES* tab.